pkg/adaptor/cloud/libvirt/libvirt.go

```diff
@@ -316,19 +354,21 @@ func createDomainXMLs390x(client *libvirtClient, cfg *domainConfig) (*libvirtxml
 				},
 			},
 		},
-	}, nil
+	}
+
+	return domain, nil
```

Comment:
Any specific reason to change this method other than updating the parameters to include vmConfig ?